art/classifiers/classifier.py

```diff
@@ -201,7 +201,14 @@ def _apply_preprocessing_standardisation(self, x):
         :type x: `np.ndarray`
         :return: Array for `x` with the standardized data.
         :rtype: `np.ndarray`
+        :raises: `TypeError`
         """
+        if x.dtype in [np.uint8, np.uint16, np.uint32, np.uint64]:
+            raise TypeError('This classifier is using `preprocessing` which can lead to negative values in the '
```

Comment:
nit: since moving this up it is no longer certain that `preprocessing` is being used. Likely want to re-word the errror msg

Reply:
That's true, thank you for the comment. I'll update the message accordingly.